src/components/atoms/Heading/Heading.test.tsx

```diff
@@ -19,7 +20,30 @@ describe('<Heading />', () => {
     ${'h3'}
     ${'h4'}
   `('it can render with a different HTML tag: $tag', ({ tag }: { tag: 'h1' | 'h2' | 'h3' | 'h4' }) => {
-    const { container } = render(<Heading as={tag}>Text</Heading>);
+    const { container } = render(<Heading as={tag}>Header</Heading>);
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it.each`
+    hex                     | name
+    ${colors.neutral.white} | ${'White'}
```

Comment:
I think you can have just `White` without the `${}`. Same for `Dark`

Reply:
I tried, and somehow it doesn't catch it 🤷🏻‍♂️ , I think Jest uses `${}` to grab the values from the template function 🤔